README.md

```diff
@@ -1,86 +1,99 @@
-# OpenApocalypse
- 
+# OpenApocalypse [![Linux Build Status](https://travis-ci.org/OpenApoc/OpenApoc.svg?branch=master)](https://travis-ci.org/OpenApoc/OpenApoc) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/07ndsvrbyct924a1)](https://ci.appveyor.com/project/JonnyH/openapoc)
```

Comment:
I *hope* to get the windows builds switched to the new repository soon - I won't be able to really look for a couple of days though, so removing the badge temporarily might be the best solution